meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor.bb

```diff
@@ -57,13 +59,29 @@ do_patch[noexec] = "1"
 do_configure[noexec] = "1"
 do_compile[noexec] = "1"
 
+api_fetch_supervisor_image() {
+	_version=$1
+	_slug=$(jq --raw-output '.slug' "${TOPDIR}/../${MACHINE}.json")
+	_api_env="${BALENA_API_ENV}"
+	_token="${BALENAOS_TOKEN:-""}"
+
+	curl -X GET --silent -k \
+	"https://api.balena-cloud.com/v6/supervisor_release?\$select=image_name&\$filter=(is_for__device_type/slug%20eq%20%27${_slug}%27)%20and%20(supervisor_version%20eq%20%27${_version}%27)" \
+	-H "Content-Type: application/json" \
+	-H "Authorization: Bearer \'${_token}\'" | jq -r '.d[].image_name'
```

Comment:
Not super critical. I think you don't need a token here. I'm also wondering what happens if `BALENAOS_TOKEN` is empty

Reply:
I tested the empty `BALENAOS_TOKEN` and it has no impact on the query

Reply:
hi @pipex, you're right, I'll clean up the token from this call.